app/components/EquivalentCliCommand.tsx

```diff
@@ -31,7 +31,12 @@ export function EquivalentCliCommand({ command }: { command: string }) {
 
   return (
     <>
-      <Button variant="ghost" size="sm" className="ml-2" onClick={() => setIsOpen(true)}>
+      <Button
+        variant="ghost"
+        size="sm"
+        className="md-:hidden"
```

Comment:
Can be disabled universally I think. Users will not be using the CLI from their phones.